nym-node-status-api/nym-node-status-api/src/http/models.rs

```diff
@@ -419,18 +420,19 @@ fn calculate_score(gateway: &Gateway, probe_outcome: &LastProbeResult) -> ScoreV
                 0.1
             };
 
-            // combine the scores
-            file_download_score * ping_ips_performance
+            (file_download_score, ping_ips_performance)
         })
-        .unwrap_or(0f64);
+        .unwrap_or((0.0, 0.0));
```

Comment:
I'd say define a helper struct, like 
```rust
struct NodeScore {
  download_speed_score: f64,
  ping_ips_score: f64
}
```

rather than passing an (f64, f64) tuple, because somebody, at some point, will accidentally mix them up